ftests/nuxeo-platform-error-web-tests/src/test/resources/log4j2-server-test.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+  <Appenders>
+    <File name="FILE" fileName="${sys:nuxeo.log.dir}/server.log" append="false">
+      <ThresholdFilter level="DEBUG" onMatch="ACCEPT" />
+      <PatternLayout pattern="%d{DEFAULT} [%t] %-5p [%C{1}] %m%n" />
+    </File>
+  </Appenders>
+  <Loggers>
+    <!-- Tests cause error logs on loggers below -->
```

Comment:
I agree it's more elegant to use a log42 conf to avoid adapting the findText, but ultimately, what we want with these ftests is to make sure error management is accurate, so checking for the presence of these ERROR logs would be better.
This is just for refinement, so I'm ok if you just keep this in mind for when such tests are moved to a non-deprecated repository.